members_and_sponsors.md

```diff
@@ -36,3 +36,4 @@ Members with affiliations:
 - Hyukjin Kwon - Apache Spark, Koalas
 - Takuya Ueshin - Apache Spark, Koalas
 - Xiao Li - Apache Spark
+- Leo Fang - Brookhaven Nat'l Lab, CuPy, mpi4py
```

Comment:
We'd like to add only names of array/dataframe libraries and of sponsoring companies here. Could you limit it to `CuPy`?

Reply:
Ah OK, sorry Ralf, didn't know this expectation. I just updated. (Though my developer time is partially on federal funding and it'd be nice to have a way to acknowledge it 🙂)

Reply:
Any form is good to me 🙂 It could also be a short session down below sponsors on the landing page of data-apis.org.